cli/src/main/java/com/devonfw/tools/ide/context/AbstractIdeContext.java

```diff
@@ -841,6 +842,7 @@ private ValidationResult applyAndRun(CliArguments arguments, Commandlet cmd) {
       debug("Running commandlet {}", cmd);
       if (cmd.isIdeHomeRequired() && (this.ideHome == null)) {
         throw new CliException(getMessageIdeHomeNotFound(), ProcessResult.NO_IDE_HOME);
+      } else if (cmd.isIdeRootRequired() && (this.ideRoot == null)) {
```

Comment:
Did you duplicate this entry by accident? Or maybe commited this change by accident but had it as a workaround before overriding `isIdeRootRequired()`?
```suggestion
```